Sources/MountebankSwift/Models/Stub/Response/Is/Is.swift

```diff
@@ -1,20 +1,19 @@
 import Foundation
 
 // swiftlint:disable type_name
-/// The `Is` response type represents a canned response that you define.The response
-/// fields will be those defined with each protocol.
+/// A regular predefined response. Merges the specified response fields with ``Imposter``.`defaultResponse`
 public struct Is: StubResponse, Equatable {
     public static var defaultBehaviors: [Behavior] = []
     public static var defaultHeaders: [String: String] = [:]
```

Comment:
Hmm, I wonder if we actually need these two static defaults ^. I think you could also just put these defaults on the DefaultResponse. But the downside would be that they would also be used for when MB does not match anything and a 404 is returned. But on the other hand, you want to eliminate those in your test anyway. I'll check if I can remove these two later.

Reply:
Indeed, they could be removed later on.